charts/seed-bootstrap/templates/gardener-external-admission-controller.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gardener-external-admission-controller-tls
+  namespace: garden
+type: kubernetes.io/tls
+data:
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVSRENDQXl5Z0F3SUJBZ0lVYlBtK3NXanI1TXhhSlREemxudGNqZ1FOSGIwd0RRWUpLb1pJaHZjTkFRRUwKQlFBd1NERUxNQWtHQTFVRUJoTUNWVk14Q3pBSkJnTlZCQWdUQWtOQk1SWXdGQVlEVlFRSEV3MVRZVzRnUm5KaApibU5wYzJOdk1SUXdFZ1lEVlFRREV3dGxlR0Z0Y0d4bExtNWxkREFnRncweE9EQTVNall4TURVM01EQmFHQTh5Ck1URTRNRGt3TWpFd05UY3dNRm93UlRFTE1Ba0dBMVVFQmhNQ1ZWTXhDekFKQmdOVkJBZ1RBa05CTVJZd0ZBWUQKVlFRSEV3MVRZVzRnUm5KaGJtTnBjMk52TVJFd0R3WURWUVFERXdobllYSmtaVzVsY2pDQ0FTSXdEUVlKS29aSQpodmNOQVFFQkJRQURnZ0VQQURDQ0FRb0NnZ0VCQUxzUEJidzQxbnJkVkpwaTFQV2ZXVEZkYktnRmlrR2w5N2FUCnJZQUN0bUVYdWlhMzdja2k2SGVwR0R3SlNwc3ZUcWsrbFZJdHgxaERlMTlRQkxheHpBSTJqNjZIUmt6aWY3czQKL3NlaWg2OHR5RnJ1QXdsbXpoekEzbThWZ1hISm9QSVdEeEZ3YVJCR1paMmVTZzJ5Uk1nU0xIeTB6K1loaG5CWApwd0FQd05OdU5pNEJDZm4zZ3pFeXdzNGd6amozMSsxNkNDQzBGdzcyT3ZNdW1aRlhhOTZnYUEyNUNrdDRPWXJlCmdmaU0wNnNtek1zQlRPRnNaMzhjN1FqMWlBbUtESytObWkxMVB4UittcGkrYXQza2thakY3SS9GcTdWT05reEsKV1lHVWIxS2NqQm9BOVR2U3BvRkx3cnFOTENjUGtld3E0c2pSN0xQMzU2OWdweSt1Y0ZNQ0F3RUFBYU9DQVNVdwpnZ0VoTUE0R0ExVWREd0VCL3dRRUF3SUZvREFUQmdOVkhTVUVEREFLQmdnckJnRUZCUWNEQVRBTUJnTlZIUk1CCkFmOEVBakFBTUIwR0ExVWREZ1FXQkJSRVQvL3dYUUVyNGhJUzE4V3E5WGpKekNyaEd6QWZCZ05WSFNNRUdEQVcKZ0JUV2FCWnpyblBwZVBTQ092Nm9udkFJcFk3UGx6Q0Jxd1lEVlIwUkJJR2pNSUdnZ2labllYSmtaVzVsY2kxbAplSFJsY201aGJDMWhaRzFwYzNOcGIyNHRZMjl1ZEhKdmJHeGxjb0l0WjJGeVpHVnVaWEl0WlhoMFpYSnVZV3d0CllXUnRhWE56YVc5dUxXTnZiblJ5YjJ4c1pYSXVaMkZ5WkdWdWdqRm5ZWEprWlc1bGNpMWxlSFJsY201aGJDMWgKWkcxcGMzTnBiMjR0WTI5dWRISnZiR3hsY2k1bllYSmtaVzR1YzNaamdoUm9iM04wTG1SdlkydGxjaTVwYm5SbApjbTVoYkRBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQWtqQ05zaTdwcjZiYmFncUcwUVFyZllPeEhpZWlwYmdMCnYzK3Zuc012V2xHSWJweGM4c2RxdVpxYkpob2NFOGUzbGF4NnNCeW5WZG9Geld6VlhhRDhaWHNmU092V0w2ZXUKWnZuZklFN2laUVkxUERldlIyN2dvTE0xY2ZPQTJJM0FxalI4eng0ZUVJQnlNenBTeElseWJLaHRtcTJtOTNuVwpick1nVUFuR2pvbEtmTEt6QnZaUU85RVFRbXhETy9kTG5UQ3JyaHg2cU00NWk5b0lrOWszMDZOQmtoU3FtelpNCmFvUDg4OGtkTDhWQ1BxYUNoVVZvUHg2R3JpSnZiNnROcm01cWFzTEZWOU1ONE5rUGlJZ1lpSlVKdGZBUEpiOWoKMlhibDNKU0lHWElrWW1NZDJTVE5sM0FXa3pmWGM5TVlqVTgxK3czKzFiM2lzM1Q3MjRzMVBnPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
Hard-coding those certificates is a bad practice.

Reply:
We decided to go this way because this component shall be removed as soon as possible. We don't want to maintain it. We don't want to generate any custom certificate for it. We simply want to keep the effort as minimal as possible. The certificates are not used for any authentication/authorization, just for establishing the TLS communication.